s3/sort.bal

```diff
@@ -14,24 +14,68 @@
 // specific language governing permissions and limitations
 // under the License.
 
-final string[] alphabet = ["a", "b", "c", "d", "e", "f", "g", "h", "i", "j", "k", "l", "m", "n", "o", "p", "q", "r", "s",
-"t", "u", "v", "w", "x", "y", "z"];
 const string NON_ALPHABET = "nonAlp";
 
 class SortBucket {
-
+    private final string[] alphabet = ["a", "b", "c", "d", "e", "f", "g", "h", "i", "j", "k", "l", "m", "n", "o", "p", "q", "r", "s", 
+"t", "u", "v", "w", "x", "y", "z"];
     string[] items = [];
     int index = 0;
     string[] result = [];
```

Comment:
Shall we make other class variables also `private`?

Reply:
Fixed in https://github.com/ballerina-platform/module-ballerinax-aws.s3/commit/f165d4bb6675e836e59f71c1a2a09a5ebe95c1c8